apiserver/facades/controller/caasfirewaller/firewaller.go

```diff
@@ -25,19 +29,42 @@ import (
 type ApplicationService interface {
 	// GetApplicationLife looks up the life of the specified application.
 	GetApplicationLife(context.Context, string) (life.Value, error)
+
 	// GetUnitLife looks up the life of the specified unit.
 	GetUnitLife(context.Context, string) (life.Value, error)
+
+	// GetApplicationIDByName returns a application ID by application name. It
+	// returns an error if the application can not be found by the name.
+	GetApplicationIDByName(ctx context.Context, name string) (application.ID, error)
```

Comment:
Shouldn't this refer to UUID rather than ID?

Reply:
Yes renaming, this is a task in @wallyworld's backlog I believe